lib/streamaggr/total.go

```diff
@@ -134,12 +134,14 @@ func (as *totalAggrState) removeOldEntries(currentTime uint64) {
 		if deleted {
 			// Mark the current entry as deleted
 			sv.deleted = deleted
+			ctx.a.staleOutputSamples.Inc()
```

Comment:
@AndrewChubatiuk I think we should make local variables and increment them instead. Then, after m.Range finishes - we update both counters via .Add method. This should reduce probability of slowdown of delete loop in case if many series became stale at once.
Could you please update it here and in other outputs?

Reply:
updated